### PR TITLE
Remove ipp feature flag

### LINKED ISSUE #6667: Remove IPP feature flag

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #6667

### Description

Removes unused IPP feature flag

### Testing instructions

Run IPP flows in debug and Release build and

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/support/HelpActivity.kt [hunk 1]
```diff
@@ -63,7 +62,7 @@ class HelpActivity : AppCompatActivity() {
         }
 
         with(binding.contactPaymentsContainer) {
-            visibility = if (FeatureFlag.CARD_READER.isEnabled()) View.VISIBLE else View.GONE
+            visibility = View.VISIBLE
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/support/HelpActivity.kt]: ❓ Do we need to explicitly set the visibility for the view here?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/MainSettingsFragment.kt [hunk 2]
```diff
@@ -239,8 +252,7 @@ class MainSettingsFragment : Fragment(R.layout.fragment_settings_main), MainSett
     }
 
     private fun updateStoreSettings() {
-        binding.storeSettingsContainer.visibility =
-            if (CARD_READER.isEnabled()) View.VISIBLE else View.GONE
+        binding.storeSettingsContainer.visibility = View.VISIBLE
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/MainSettingsFragment.kt]: ❓ Do we need to explicitly set the visibility for the view here?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt [hunk 3]
```diff
@@ -22,8 +21,7 @@ enum class FeatureFlag {
                 PackageUtils.isDebugBuild() || context != null && PackageUtils.isBetaBuild(context)
             }
             JETPACK_CP,
-            IN_PERSON_PAYMENTS_CANADA,
-            CARD_READER -> true // Keeping the flag for a few sprints so we can quickly disable the feature if needed
+            IN_PERSON_PAYMENTS_CANADA -> true
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt:25]: 🔍 ❓ Do you think referencing [this](https://github.com/woocommerce/woocommerce-android/issues/6584) issue here for `IN_PERSON_PAYMENTS_CANADA` flag and retaining the comment `Keeping the flag for a few sprints so we can quickly disable the feature if needed` makes sense?

**Review by AnirudhBhat — APPROVED** [review]
Thank you for working on this @kidinov I've tested the changes and it is working as expected.  I've left a few minor questions and they are not blocking for merge.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/support/HelpActivity.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/support/HelpActivity.kt]: Yep, done here - d2ebada3b1843f768931a0e6e07a785cff351412

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/MainSettingsFragment.kt (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/prefs/MainSettingsFragment.kt]: Yep, done here - d2ebada3b1843f768931a0e6e07a785cff351412

File: WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt (hunk 3 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt:25]: Moved ht comment close to the canada flag - d2ebada3b1843f768931a0e6e07a785cff351412

**Pull request merged**